pyigm/surveys/dlasurvey.py

```diff
@@ -164,33 +168,41 @@ def load_H100(cls, grab_spectra=False, load_sys=True, isys_path=None):  #skip_tr
             # Load ions
             dla_survey.fill_ions(jfile=ions_fil)
 
-        dla_survey.ref = 'Neeleman+13'
-
-        """
-        # Load transitions
-        names = list(dla_survey.name)
-        if not skip_trans:
-            print('H100: Loading transitions file {:s}'.format(trans_fil))
-            tar = tarfile.open(trans_fil)
-            for member in tar.getmembers():
-                if '.' not in member.name:
-                    print('Skipping a likely folder: {:s}'.format(member.name))
-                    continue
-                # Extract
-                f = tar.extractfile(member)
-                # Need to fix for 3.4
-                tdict = json.load(f)
-                # Find system
-                i0 = member.name.rfind('/')
-                i1 = member.name.rfind('_clm')
-                try:
-                    idx = names.index(member.name[i0+1:i1])
-                except ValueError:
-                    pdb.set_trace()
-                # Fill up
-                dla_survey._abs_sys[idx].load_components(tdict)
-        """
+        # Metallicities
+        tbl2_file = resource_filename('pyigm', "/data/DLA/H100/H100_table2.dat")
+        tbl2 = Table.read(tbl2_file, format='cds')
+        names = dla_survey.name
+        qsonames = []
+        zabs = []
+        for name in names:
+            prs = name.split('_')
+            qsonames.append(prs[0])
+            zabs.append(float(prs[1][1:]))
+        qsonames = np.array(qsonames)
+        zabs = np.array(zabs)
+        for ii, iqso, izabs in zip(range(len(tbl2)), tbl2['QSO'], tbl2['zabs']):
+            mt = np.where((qsonames == iqso) & (np.abs(izabs-zabs) < 1e-3))[0]
+            if len(mt) == 0:
+                pdb.set_trace()
+            elif len(mt) != 1:
+                pdb.set_trace()
+            # Metallicity
+            dla_survey._abs_sys[mt[0]].ZH = tbl2['[M/H]'][ii]
+            dla_survey._abs_sys[mt[0]].sig_ZH = tbl2['e_[M/H]'][ii]
+            if tbl2['M'][ii] in ['S','Si','O']:
```

Comment:
Am trying to make it somewhat backwards compatible to
what we had in the old days..  

But good point.  Am now also saving the actual element too.